pkg/printer/README.md

```diff
@@ -0,0 +1,42 @@
+# printer
```

Comment:
Love having READMEs and usage instructions for packages! We should adopt this for all new packages written for sysl!

Reply:
👍